Tests/Test_ReadCarCANrewrite.c

```diff
@@ -0,0 +1,195 @@
+/**
+ * 
+ * This file tests ignition and charge enable interactions.
+ * 
+ *
+ * This test is run in LoopBack mode with all messages sent and received by the LeaderBoard.
+ * However, it can be run in conjunction with motor-sim and car-sim
+ * (which don't do anything) when simulated to appease Renode.
+ * 
+ * @file 
+ * @author Diya Rajon (drajon@utexas.edu)
+ * @brief Tests read car can structure
+ * @version 2
+ * @date 2023-7-14
+ *
+ * @copyright Copyright (c) 2022 Longhorn Racing Solar
+ *
+ */
+
+#include "Tasks.h"
+#include "CANbus.h"
+#include "CAN_Queue.h"
+#include "ReadCarCAN.h"
+#include "Contactors.h"
+#include "Display.h"
+#include "UpdateDisplay.h"
+#include "CANConfig.h"
+
+static OS_TCB Task1_TCB;
+#define STACK_SIZE 128
```

Comment:
Just use DEFAULT_STACK_SIZE defined in Tasks.h. @Cam0Cow also mentioned that setting STACK_SIZE to 128 causes problems especially when the stack becomes too big.